src/msix/IXml.cpp

```diff
@@ -24,7 +24,7 @@ static const char* attributeNames[] = {
     /* Language                               */"Language",
     /* MinVersion                             */"MinVersion",
     /* Dependencies_Tdf_MaxVersionTested      */"MaxVersionTested",
-    /* Package_Applications_Application_Id    */"Id",

```

Comment:
Bad merge of this; the value has been lost.  We should have a static_assert it would seem.

Reply:
Should I put the value back in?